src/main/scala/xiangshan/frontend/bpu/ittage/Parameters.scala

```diff
@@ -51,6 +52,10 @@ trait HasIttageParameters extends HasBpuParameters {
   def TableInfos: Seq[IttageTableInfo] = ittageParameters.TableInfos
   def NumTables:  Int                  = TableInfos.length
 
+  def IttageNumBanks:     Int = ittageParameters.NumBanks
+  def IttageBankIdxWidth: Int = log2Ceil(IttageNumBanks)
```

Comment:
```suggestion
  def NumBanks:     Int = ittageParameters.NumBanks
  def BankIdxWidth: Int = log2Ceil(NumBanks)
```

`trait HasIttageParameters` is designed to be used only in ittage, so here no `Ittage` prefix is needed.